build/Dockerfile.ci

```diff
@@ -21,39 +21,29 @@ RUN echo "4a7fa60f323ee1416a4b1425aefc37ea359e9d64df19c326a58953a97ad41ea5  go.t
 RUN tar -C /usr/local -xzf go.tar.gz
 ENV PATH=${PATH}:/usr/local/go/bin
 
+# Build WMCO
+# The source here corresponds to the code in the PR and is placed here by the CI infrastructure.
+RUN mkdir /build/windows-machine-config-operator
+WORKDIR /build/windows-machine-config-operator
+COPY . .
+RUN make build
+
```

Comment:
Why is this being moved?

Reply:
moving the copying of WMCO before building submodule binaries since `COPY . .` will copy all the submodules  and we would only need to change the directory and build them.